src/components/Page-Title/Page-Title.js

```diff
@@ -0,0 +1,73 @@
+import * as PropTypes from 'prop-types';
+import * as React from 'react';
+
+export default class PageTitle extends React.Component {
+  render() {
+    let help = '';
+    let link = '';
+    let subHeader = '';
+
+    if (this.props.helpIcon) {
+      help = (
```

Comment:
Looks like help is defined as an empty string (`let help = '';`).
Here we are reasigning its value as JSX.Element which will throw an error once we migrate to TypeScript.

You could use:
```
const help = this.props.helpIcon ?
    (
      <a
            className="chi-button -icon -flat -lg -bg--none -b--transparent -opacity-hover--80"
            onClick={this.props.helpclick}
            onMouseEnter={this.props.mouseoverhelp}
            onMouseLeave={this.props.mouseleavehelp}>
            <div className="chi-button__content">
              <i className="chi-icon icon-circle-question-outline"></i>
            </div>
          </a>
    )
  : null;
```

The same for link, subheader, etc

Reply:
changes added